enzyme/Enzyme/MLIR/Dialect/EnzymeOps.td

```diff
@@ -222,4 +222,11 @@ def SampleOp : Enzyme_Op<"sample",
   }];
 }
 
+def DetachGradientOp : Enzyme_Op<"detach_gradient",
+    [Pure, SameOperandsAndResultElementType, SameOperandsAndResultShape]> {
+  let summary = "Prevents the flow of gradients by creating a new tensor that is detached from the original tensor.";
```

Comment:
Replace value with tensor. And maybe specify that this is an identity operation in the primal , and that this block blocks all higher order derivatives as well